src/test/java/com/google/api/codegen/testdata/py/python_no_path_templates.baseline

```diff
@@ -970,9 +942,110 @@ config = {
 # limitations under the License.
 
 
+import google.api_core.grpc_helpers
+
+from example.proto import no_path_templates_pb2_grpc
+
+
+class NoTemplatesApiServiceGrpc(object):
```

Comment:
GrpcTransport?

Reply:
@landrito Could you make this happen?

Reply:
Bump on this, I still think it should be `GrpcTransport`.